src/sfeTk/sfeTkIBus.h

```diff
@@ -87,14 +87,32 @@ class sfeTkIBus
 {
   public:
     /**--------------------------------------------------------------------------
-     *  @brief Write a single byte to the device*
+     *  @brief Send a single byte to the device*
```

Comment:
Since all the methods use a "write*" naming pattern, shouldn't this be Write? 

FWIW: Normally send/receive are used with network based operations - at least in my experience ....

Reply:
You're completely right. I was trying to get at the fact that it doesn't physically write bits/bytes to a register and so "send" seemed to fit. However, you're write. I'll just clarify that we're "Write a single byte, without indexing to a register". Something to that effect. I also have a few more things that I want to add to this pull request.